pkg/arvo/sys/vane/gall.hoon

```diff
@@ -1906,6 +2053,21 @@
   =/  =coin  $/r.bem
   =*  path  s.bem
   ::
+  ?:  ?=(%a care)
+    ?.  =(p.bem our)  ~
+    =/  yok  (~(get by yokes.state) q.bem)
+    ?~  yok  ~
+    =/  ski  (~(get by sky.u.yok) s.bem)
+    ?~  ski  ~
+    =/  res=(unit (each noun @uvI))  
+      ?+  -.r.bem  ~
+        %ud  (get:on-path fan.u.ski p.r.bem)
+        %da  ?>(=(p.r.bem now) (bind (ram:on-path fan.u.ski) tail))
```

Comment:
This shouldn't assert, it should block (`~`) at dates other than `now`. (Alternately, we could store the dates on the entries and support other `%da` cases.)